docs/reference/mapping/types/geo-shape.asciidoc

```diff
@@ -100,16 +107,33 @@ and reject the whole document.
 
 |=======================================================================
 
+
+[[indexing-approach]]
```

Comment:
we need a more qualified name since this links may be used from anywhere in the docs, eg. `geo-shape-indexing-approach`